src/App.php

```diff
@@ -911,7 +911,7 @@ public function getTag($tag = null, $attr = null, $value = null)
             $tag = substr($tag, 0, -1);
             $postfix = '/';
         } elseif (substr($tag, 0, 1) == '/') {
-            return isset($attr[0]) ? '</' . $attr[0] . '>' : '<' . $tag . '>';
+            return '</' . ($attr[0] ?? substr($tag, 1)) . '>';
```

Comment:
This is changing (wrong?) change at all. What if $attr[0] wasn't defined? Previously it output `<$tag>`, now it will be `</$tag>` !? isn't it?

Reply:
Can you provide a test code?

Reply:
@mvorisek ^^ this seems broken. Can we revert this bit here? currently it should return '<tag>' but in your case '</..' is always returned.

Reply:
oh i didn't notice the substr, so this should be ok !!

Reply:
I can patch it if needed, what is the issue, can you provide a simple code/data to test the issue?

Reply:
@romaninsh Yes, seems much better to me, I hate defining anything twice. :D